java/code/src/com/redhat/rhn/frontend/action/common/DownloadFile.java

```diff
@@ -641,6 +641,9 @@ else if (actualFile.exists()) {
             }
         }
         log.debug("Final path before returning getStreamForPath(): {}", diskPath);
+        if (log.isDebugEnabled()) {
+            log.debug("Final path before returning getStreamForPath(): {}", diskPath);
+        }
```

Comment:
Please remove this change: it's useless to have that `log.debug()` in an `if` branch and is a duplicate with the line above